lib/engine_http.js

```diff
@@ -34,13 +34,13 @@ function HttpEngine(config) {
   this.config = config;
 }
 
-HttpEngine.prototype.step = function step(requestSpec, ee) {
+HttpEngine.prototype.step = function step(requestSpec, scenarioSpec, ee) {
```

Comment:
There is no need to pass a whole scenario spec in here, if `followRedirect` false is set in the requestSpec already it will just work.